app/assets/stylesheets/companies/companies_index.scss

```diff
@@ -60,7 +60,8 @@ $font-company-2: "metropolisblack",sans-serif;
       position: relative;
       background-color: #d9d9d9;
       height: 300px;
-      width: 250px;
+      // width: 250px;
```

Comment:
please, delete this line, if you don't need it

Reply:
fixed